.github/ISSUE_TEMPLATE/tracking.yml

```
@@ -0,0 +1,101 @@
+      description: Any other information relevant to tracking this system
+      placeholder: "Timeline considerations, resource requirements, etc..."
+    validations:
+      required: false
+  - type: input
+    id: meme-upload
+    attributes:
+      label: Mandatory Meme Upload
+      description: Please upload a relevant (or completely random) meme
+      placeholder: "Drag and drop your meme image here or click to upload"
+    validations:
+      required: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)